app/javascript/chat/ChatChannelSettings/ChatChannelSettings.jsx

```diff
@@ -308,14 +308,20 @@ export default class ChatChannelSettings extends Component {
     const { currentMembership } = this.state;
     if (actionStatus) {
       const response = await leaveChatChannelMembership(currentMembership.id);
+      const { message } = response;
       if (response.success) {
+        this.setState({
+          successMessages: response.message,
```

Comment:
You destructure `response` a few lines above (`const { message } = response;`), shouldn't you be using `message` here?

```suggestion
          successMessages: message,
```

Reply:
That makes sense!

This isn't actually my code (it's a rebase of a stale PR), so I'm not sure, I'll test that suggestion and commit it if it works as expected.